src/core/issues-notes.js

```diff
@@ -146,42 +137,46 @@ function handleIssues(ins, ghIssues, conf) {
             })
             .map(createLabel)
             .reduce((frag, labelElem) => {
-              frag.appendChild(labelElem);
+              frag.append(labelElem);
               return frag;
             }, document.createDocumentFragment());
-          $tit.append(
-            $(
-              "<span style='text-transform: none'>: " + report.title + "</span>"
-            ).append(labelsGroup)
-          );
-          $inno.removeAttr("title");
+          const node = hyperHTML`<span style='text-transform: none'>: ${
+            report.title
+          }</span>`;
+          node.append(labelsGroup);
+
+          tit.append(node);
+          inno.removeAttribute("title");
         } else if (report.title) {
-          $tit.append(
-            $(
-              "<span style='text-transform: none'>: " + report.title + "</span>"
-            )
+          tit.append(
+            hyperHTML`<span style='text-transform: none'>: ${
+              report.title
+            }</span>`
           );
-          $inno.removeAttr("title");
+          inno.removeAttribute("title");
         }
-        $tit.addClass("marker");
-        $div.append($tit);
-        $inno.replaceWith($div);
-        let body = $inno.removeClass(report.type).removeAttr("data-number");
-        if (ghIssue && !body.text().trim()) {
-          body = ghIssue.body_html;
+        tit.classList.add("marker");
+        div.append(tit);
+        let body = inno;
+        inno.replaceWith(div);
+        body.classList.remove(report.type);
+        body.removeAttribute("data-number");
+        if (ghIssue && !body.innerHTML.trim()) {
+          body = hyperHTML`${ghIssue.body_html}`;
         }
-        $div.append(body);
-        const level = $tit.parents("section").length + 2;
-        $tit.attr("aria-level", level);
+        div.append(body);
+        const level = getCount(tit);
```

Comment:
We recently added `util.parents()` function to do this. So you may do `parents(tit, "section") + 2`.

Reply:
gotcha. Thanks.